Dockerfile

```diff
@@ -31,6 +32,10 @@ RUN yarn add puppeteer-core@chrome-$(chromium-browser --version | sed -r 's/^Chr
 RUN yarn install && yarn build && rm -rf ./src ./node_modules && yarn install --production && yarn cache clean \
     && node /home/marp/.cli/marp-cli.js --version
 
+USER root
+
+ENV MARPID marp:marp
```

Comment:
I felt the env name `MARPID` is unclear and a bit obscure. How about using `MARP_USER` instead?

Reply:
Definitely better.